target/linux/ath79/mikrotik/base-files/etc/board.d/02_network

```diff
@@ -7,6 +7,25 @@ ath79_setup_interfaces()
 	local board="$1"
 
 	case "$board" in
+	mikrotik,routerboard-2011il|\
+	mikrotik,routerboard-2011l)
+		ucidef_add_switch "switch0" \
+			"0@eth0" "2:lan" "3:lan" "4:lan" "5:lan" "1:wan"
+		ucidef_add_switch "switch1" \
+			"0@eth1" "1:lan:5" "2:lan:4" "3:lan:3" "4:lan:2" "5:lan:1"
+		;;
+	mikrotik,routerboard-2011ils|\
+	mikrotik,routerboard-2011ls|\
+	mikrotik,routerboard-2011uas|\
+	mikrotik,routerboard-2011uas-2hnd|\
+	mikrotik,routerboard-2011uias|\
+	mikrotik,routerboard-2011uias-2hnd)
+		ucidef_add_switch "switch0" \
+			"0@eth0" "2:lan" "3:lan" "4:lan" "5:lan" "1:wan" "6:sfp"
```

Comment:
can we add sfp to lan or wan group? right now it's not usable by default.

Reply:
I don't see why not. AFAIK, the right way would be to replicate the default configuration from RouterOS, do you know if SFP is part of the WAN group/bridge there? I'm not able to check it at the moment.